sdcm/sct_config.py

```diff
@@ -1845,7 +1845,9 @@ def _validate_value(self, opt):
                     and self.get("k8s_tenants_num") > 1
                     and opt.get("k8s_multitenancy_supported")
                     and isinstance(self.get(opt['name']), list)):
-                raise ValueError("failed to validate {}".format(opt['name'])) from ex
+                if not (isinstance(self.get('nemesis_class_name'), str)
+                        and len(self.get("nemesis_class_name").split(" ")) > 1):
```

Comment:
Actually, you have bug here.
If `nemesis_class_name` is set to be string and has multiple words separated with space then any other broken option won't be caught.

It must look like something following:
```
if not (
    opt['name'] == 'nemesis_class_name' 
    and isinstance(self.get(opt['name']), str) 
    and len(self.get("nemesis_class_name").split(" ")) > 1):
```

Reply:
fixed